ieeg_MED/dataset_description.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Name": "Brainstorm epileptogenicity tutorial",
+    "BIDSVersion": "1.7.0",
+    "License": "This tutorial dataset (EEG and MRI data) remains property of the Grenoble University Hospital, France. Its use and transfer outside the ImaGIN tutorial, e.g. for research purposes, is prohibited without written consent. For questions, please contact Olivier David, PhD (olivier.david@inserm.fr).",
+	 "Authors": [
```

Comment:
would also make sense to update the readme because of all the authors and institutions metadata: these people may otherwise wonder what dataset of theirs this one is linked to

Reply:
feels easier than cleaning up all the dataset of any reference of the dataset it was copied from